message/models.py

```diff
@@ -0,0 +1,15 @@
+from django.db import models
+from player.models import Player
+from game_event.models import GameEvent
+
+# Create your models here.
+
+
+class Message(models.Model):
+    user_id = models.ForeignKey(Player, on_delete=models.CASCADE)
+    game_event_id = models.ForeignKey(GameEvent, on_delete=models.CASCADE)
+    time_sent = models.DateTimeField(auto_now_add=True)
+    message_text = models.CharField(max_length=255)
+
+    class Meta:
+        unique_together = ('game_event_id', 'user_id')
```

Comment:
What is the purpose of this?
Does it imply that every user is restricted to sending just one message per event?

Reply:
Well there is the dateTime field there, so each messeage have unique dateTime, i am guessing that you can't send 2 messeages quick enough to make both have the same dateTime object, so it could work as it is right now.

Reply:
the unique together function is for sets of field names that, taken together, must be unique.
have nothing to do with 2 foreign keys.
see more about unique-together: https://docs.djangoproject.com/en/3.1/ref/models/options/#unique-together

Reply:
@Yuval-Vino we removed the unique together function from our model file.